docs/modules/operation/pages/flows/basic.adoc

```diff
@@ -0,0 +1,142 @@
+** (optional) Configure Elasticsearch variables like `search.max_buckets` if the default values are not sufficient.
+** (optional) Create a job to clean the indices so that the disk does not fill up; for example, keep X days of flows.
+Filled disks are a challenging problem to address for non-Elasticsearch experts.
+We recommend the Elasticsearch https://www.elastic.co/guide/en/elasticsearch/client/curator/current/index.html[Curator tool] to do this.
+** Monitor the Elasticsearch stack in OpenNMS to get an alarm if it goes down.
+* Set up OpenNMS Helm with Grafana to visualize flows
+** Configure the flow and performance data sources
+
+== Configure OpenNMS to communicate with Elasticsearch
+
+OpenNMS needs to talk to Elasticsearch and know where to store the flows data it collects (persistence).
+
+From a Karaf shell on your {page-component-title} instance, update `$\{OPENNMS_HOME}/etc/org.opennms.features.flows.persistence.elastic.cfg` to configure the flow persistence to use your Elasticsearch cluster:
+
+.Connect to Karaf shell
+[source, console]
+----
+ssh -p 8101 admin@localhost
+----
+
+.Configure Elasticsearch settings within Karaf
+[source, karaf]
+----
+config:edit org.opennms.features.flows.persistence.elastic
+config:property-set elasticUrl http://elastic:9200
+config:update
+----
+
+We also recommend setting the following:
+
+.Edit (or create) `$\{OPENNMS_HOME}/etc/org.opennms.features.flows.persistence.elastic.cfg`
+[source, xml]
+----
+# ElasticSearch persistence configuration
+elasticUrl = http://10.10.3.218:9200 <1>
+connTimeout = 30000
+readTimeout = 300000
+settings.index.number_of_replicas = 0
+settings.index.number_of_shards=1
+settings.index.refresh_interval=10s
+elasticIndexStrategy=daily
+----
+<1> Replace with comma-separated list of Elasticsearch nodes.
+
+See <<elasticsearch/introduction.adoc#ga-elasticsearch-integration-configuration, General Elasticsearch Configuration>> for a complete set of options.
+
+== Enable protocols
+
+Update `$\{OPENNMS_HOME}/etc/telemetryd-configuration.xml` to enable one or more of the protocols you want to handle.
+
+This example enables the NetFlow v5 protocol.
+Use the same process for any of the other flow-related protocols.
+
+[source, xml]
+----
+<listener name="Netflow-5-UDP-8877" class-name="org.opennms.netmgt.telemetry.listeners.UdpListener" enabled="true">
+    <parameter key="port" value="8877"/>
+
+    <parser name="Netflow-5-Parser" class-name="org.opennms.netmgt.telemetry.protocols.netflow.parser.Netflow5UdpParser" queue="Netflow-5" />
+</listener>
+
+<queue name="Netflow-5">
+    <adapter name="Netflow-5-Adapter" class-name="org.opennms.netmgt.telemetry.protocols.netflow.adapter.netflow5.Netflow5Adapter" enabled="true">
+    </adapter>
+</queue>
+----
+
+Send a `reloadDaemonConfig` event via the CLI to apply the changes without restarting:
+
+[source, console]
+----
+$\{OPENNMS_HOME}/bin/send-event.pl -p 'daemonName Telemetryd' uei.opennms.org/internal/reloadDaemonConfig
```

Comment:
@Bonrob2 Is that still the way we want to offer? 

@mmahacek @indigo423 I've never tried it but can't we do something like:

`ssh -p 8101 admin@localhost 'reload-daemon telemetryd'`